presto-docs/src/main/sphinx/optimizer/pushdown.rst

```diff
@@ -15,22 +16,21 @@ The results of this pushdown can include the following benefits:
 Support for pushdown is specific to each connector and the relevant underlying
 database or storage system.
 
-Analysis and Confirmation
--------------------------
+Aggregation Pushdown
+--------------------
 
-Pushdown depends on a number of factors:
+Aggregation pushdown depends on a number of factors:
 
-* generic support for pushdown for that function in Presto
-* function or operation specific support for pushdown in the connector
-* query that allows the detection of the function to push down
-* function needs to exist in the underlying system so it can process the
-  pushdown
+* general support for aggregation pushdown in the connector
+* function-specific support for pushdown in the connector
+* query that allows the pushdown to take place
 
-The best way to analyze if pushdown for a specific query is performed is to
-take a closer look at the :doc:`EXPLAIN plan </sql/explain>` of the query. If an
-operation such as an aggregate function is successfully pushed down to the
-connector, the explain plan does **not** show that operator. The explain plan
-only shows the operations that are performed by Presto.
+
+The best way to analyze if aggregation pushdown for a specific query is performed is to
+take a closer look at the :doc:`EXPLAIN plan </sql/explain>` of the query. If an aggregate
+function is successfully pushed down to the connector, the explain plan does **not** show
+that ``Aggregate`` operator. The explain plan only shows the operations that are performed
+by Presto.
```

Comment:
```suggestion
You can check if pushdown for a specific query is performed by looking at the :doc:`EXPLAIN plan </sql/explain>` of the query. 
If an aggregate function is successfully pushed down to the connector, the explain plan does **not** show
that ``Aggregate`` operator. The explain plan only shows the operations that are performed by Presto.
```

Reply:
Breaking up the lines and reformatting like this does NOT affect output and violates our line length standard (80 char). Please keep it as is

Reply:
Sure, not forcing anything. Not my area of expertise.